modules/stanford_events_importer/stanford_events_importer.module

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * @file
+ * File description.
+ *
+ * Long description.
+ */
+
+use Drupal\migrate_tools\MigrateBatchExecutable;
+use Drupal\migrate\MigrateMessage;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\user\Entity\User;
+use Drupal\stanford_events_importer\StanfordEventsImporter;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Alter the config form to add the migrate_tools UI.
+ */
+function stanford_events_importer_form_config_pages_stanford_events_importer_form_alter(array &$form, FormStateInterface $form_state) {
+  $user = User::load(\Drupal::currentUser()->id());
+  $form['actions']['#type'] = 'fieldset';
+
+  $form['actions']['import'] = [
+    '#type' => 'submit',
+    '#value' => t('Save & Import'),
+    '#name' => 'op',
+    '#button_type' => "primary",
+    '#submit' => [
+      "::submitForm",
+      "::save",
+      "stanford_events_importer_form_config_pages_stanford_events_importer_form_alter_submit",
+    ],
+  ];
+
+  $form['actions']['update_opts'] = [
+    '#type' => 'submit',
+    '#value' => t('Update Org & Category Options'),
+    '#name' => 'op',
+    '#submit' => ["stanford_events_importer_update_opts"],
+    '#access' => $user->hasPermission("administer migrations"),
+  ];
+
+  return $form;
+}
+
+/**
+ * Submit handler for the config form override.
+ *
+ * @param array $form
+ *   The form array.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The form state.
+ */
+function stanford_events_importer_form_config_pages_stanford_events_importer_form_alter_submit(array &$form, FormStateInterface $form_state) {
+
+  // Reset all static caches as something is getting stuck.
+  drupal_static_reset();
+
+  // Clear all plugin caches.
+  \Drupal::service('plugin.cache_clearer')
```

Comment:
why? this gives me nervous sweats

Reply:
I was having issues on ACSF. Let me yank this and see if it is better now.